ocs_ci/helpers/helpers.py

```diff
@@ -2942,6 +2943,28 @@ def get_mon_pdb():
     return disruptions_allowed, min_available_mon
 
 
+def safe_get_attr(obj, attribute):
```

Comment:
This is clever hack :) I would suggest few tweaks:
- Rename this function to something more obvious, like: `get_attr_chain()`. Ignoring non existing attributes ans silently returning `None` is not something I would normally associate with being "safe" :)
- rename `attribute` function parameter, to `attributes` or `attr_chain` to make it obvious that you are expecting multiple attributes
- consider to change type of  the attribute parameter from string to list, that would make the API and the code of this function little bit more obvious

Reply:
Thank you very much for the comprehensive review and guidance!
This was originally a tiny internal function, thus the lacking documentation and naming. I assumed many things, most of them revolving around `cld_mgr` and the way its attributes are assigned ;)

Regarding the consideration to use a list instead of a string - I thought about it, and I felt that it'd be rather redundant to have to call `.split()` in every call to the function, instead of doing it internally.
Keeping the variable as a string also allows, in my opinion, easier code readability when used - i.e. -
`get_attr_chain(cld_mgr, "aws_client.access_key")`
as opposed to
`get_attr_chain(cld_mgr, ["aws_client", "access_key"])`
or
`get_attr_chain(cld_mgr, "aws_client.access_key".split("."))`